libraries/types/pbft_block/src/pbft_block.cpp

```diff
@@ -15,7 +15,9 @@ PbftBlock::PbftBlock(dev::RLP const& rlp) {
     dev::bytes extra_data_bytes;
     util::rlp_tuple(util::RLPDecoderRef(rlp, true), prev_block_hash_, dag_block_hash_as_pivot_, order_hash_,
                     prev_state_root_hash_, period_, timestamp_, reward_votes_, extra_data_bytes, signature_);
-    extra_data_ = PbftBlockExtraData(extra_data_bytes);
+    if (extra_data_bytes.size() > 0) {
```

Comment:
@mfrankovi is it necessary ? If the itemcount == 9, extra_data should always contain some data